go.mod

```diff
@@ -61,7 +61,6 @@ replace (
 
 require (
 	github.com/antihax/optional v1.0.0
-	github.com/containers/image/v5 v5.30.0
```

Comment:
I saw similar issues with the go mod file. We should get one the maintainers to ok this off. It does look like at least on one of the branches the mod file was behind. Can you make sure these changes are fine? I'm a bit suspicious that all of our release branches need the same go mod changes.

Reply:
Will run `go mod tidy` in all branches.

Reply:
I think this change here in `go.mod` file is related to our changes in the `pkg/image/utilities/utilities.go`.